xivo_fetchfw/package.py

```diff
@@ -75,10 +71,10 @@ def new_installed_package(self):
         return InstalledPackage(new_pkg_info)
 
     def __str__(self):
-        return "%s %s" % (self.pkg_info['id'], self.pkg_info['version'])
+        return "{} {}".format(self.pkg_info['id'], self.pkg_info['version'])
```

Comment:
Consistency: on line 96 you used the format-string form for the same kind of string.
```python3
return f"{self.pkg_info['id']} {self.pkg_info['version']}"
```